internal/ask/form/submission/submission.go

```diff
@@ -138,6 +140,41 @@ func (s *Submission) Validate() error {
 	return nil
 }
 
+// GetQuestions retrieves the questions
+func (s *Submission) GetQuestions() []string {
+	var h []string
+
+	for _, r := range s.Answers {
+		h = append(h, r.Question.(string))
+	}
+
+	return h
+}
+
+// GetAnswers get the answers on the submission and returns a slice of them
+func (s *Submission) GetAnswers() []string {
+	var v []string
+	convertToString := func(m bson.M) string {
+		var s string
+		for k, v := range m {
+			s = s + fmt.Sprintf("%v: %v ", k, v)
+		}
+		return s
+	}
+
+	// Go through the answers and convert them into a slice of strings
```

Comment:
Should also use correct grammar when adding comments (including periods and what not). I was also a bit confused by some of the code here, specifically the need to have the "convertToString" being a method defined in the function. I'd expect you could pull this out the same way you pulled out the `encodeCSV` function in the handlers.